drv/transceivers-server/src/udp.rs

```diff
@@ -304,15 +305,24 @@ impl ServerImpl {
             HostRequest::Status(modules) => {
                 ringbuf_entry!(Trace::Status(modules));
                 let mask = LogicalPortMask::from(modules);
-                let (num_status_bytes, result) = self.get_status(mask, out);
+                let (data_len, result) = if self
+                    .front_io_board_present
+                    .unwrap_or(false)
+                {
+                    self.get_status(mask, out)
+                } else {
+                    (
+                        0,
+                        ModuleResultNoFailure::new(LogicalPortMask(0), mask)
+                            .unwrap(),
```

Comment:
nit: should probably be `unwrap_lite` to avoid generating formatting code?
```suggestion
                            .unwrap_lite(),
```
(we may need to import the trait if it isn't already)